### ISSUE TITLE
Updated Climate and Policy Data and Design Notes

### ISSUE THREAD

**Issue by klongleywood** [Vizzuality/mangrove-atlas#342]
Thanks for your patience as I got this information together! Here are some notes on the climate and policy tool. I'm attaching an updated table and ppt slides to support. 

1.	FREL’s don’t warrant their own widget. Move this information to the “International Status” panel
2.	Add NDC Blurb, Climate Vulnerability Index, and Policies Relevant to Mangroves to International Status panel (Columns L, M, and Policies Tab)
3.	Under emissions, we don’t have the data to describe whether mangroves are included in GHG inventories, so take this out. Instead, on this slide, can we show the mitigation opportunity from mangroves (Columns AE and AF) against the land use emissions by category (LULC Emissions Category Tab) as a stacked bar chart or similar? The only issue is that the mangrove opportunity will almost always be tiny. We may want to play around with the idea of correcting for the % land area that the mangrove covers to try to better illustrate that opportunity. Does Vizzuality have access to land cover totals by country? If not, I can try to track this down but wondered if you had it for Global Forest Watch or similar? Maybe we can discuss options on Friday. 
4.	Note new slide on investible blue carbon and suggested chart (prefer a donut type chart like in GMW rather than pi chart shown here)

New table is[ here](https://wetlandsint.sharepoint.com/sites/GlobalMangroveWatch/_layouts/15/Doc.aspx?OR=teams&action=edit&sourcedoc={2E198C9F-7E3F-4345-AA5D-54AAEB2889AB})

New mock up is[ here](https://teams.microsoft.com/l/file/FF6BAEA9-CFB2-43E6-8B50-5EAA3B5C4A90?tenantId=26a3fd43-5ba9-415c-b904-54408786d21d&fileType=pptx&objectUrl=https%3A%2F%2Fwetlandsint.sharepoint.com%2Fsites%2FGlobalMangroveWatch%2FShared%20Documents%2FPlatform%20Team%2FPhase2_Planning%2FCarbon%20Policy%20Working%20Group%2FClimate_Policy_Widget_20211130_KLW.pptx&baseUrl=https%3A%2F%2Fwetlandsint.sharepoint.com%2Fsites%2FGlobalMangroveWatch&serviceName=teams&threadId=19:bc902c837fa04ca88e6931088cc1cf67@thread.skype&groupId=0d61d541-82c9-493d-a067-0af1f635aa72)

**Comment by tamaramegan** [comment]
Hi @klongleywood, thanks for the updates! 
I don't have access to the documents you have shared, we don't have teams accounts I am wondering if you can make the docs public or share them some other way. Thanks!

**Comment by klongleywood** [comment]
Sorry Tamara! Try https://tnc.box.com/s/gbesb7j3jn6yxeyk9ydoidgrtlcz3c82) and https://tnc.box.com/s/nrfhku0xr6ksizc7z42jaxf3gx923yfb instead.

**Comment by AngelArcones** [comment]
Hi @klongleywood,
after exploring the emissions and mitigation data, there is indeed a generalized mismatch of magnitudes, as seen in this example with the data of Indonesia:
![Emissions raw](https://user-images.githubusercontent.com/57727579/152538694-5bb40461-455f-4139-97d1-f6f8f630c8de.png) (with each bar with its own scale).
After exploring the suggested possibility of normalising by the % of area covered by mangroves, the outcome tends to be the opposite: proportion of emissions is tiny compared to mitigation values.

We would suggest another approach: normalising the emission values by the % of the country that is coastal area (considering it as a 1km buffer around coastline for test, but this should be refined). In that case the values are much more comparable: 
<img width="377" alt="Emissions corrected by coast" src="https://user-images.githubusercontent.com/57727579/152539448-aab36435-5796-4b83-a8f2-7cda58a6c890.png">

**Comment by klongleywood** [comment]
Hi @AngelArcones -- thanks for this! I brought this up with the team last week and we think that this type of approach is helpful but that it would benefit from some standardization around the various emission sources as well. I've asked my colleague to provide data on this and I expect it this week -- I should have more guidance for you on this soon.

**Comment by klongleywood** [comment]
[Copy of NCS mitigation potential density by country.xlsx](https://github.com/Vizzuality/mangrove-atlas/files/8049430/Copy.of.NCS.mitigation.potential.density.by.country.xlsx)

Hi @AngelArcones -- here is a copy for the data you can use for the bar chart on the left.  The data have both totals as well as totals normalized by land area. It might be useful to see what it looks like both normalized and non-normalized but I think we will want them normalized. It also occurs to me that for the non-mangrove interventions, if, cumulatively, they are a lot larger than the mangrove values, we could get away from the stacked bar chart and show the values as separate bars. 

Let me know if you need more info!

**Comment by AngelArcones** [comment]
Hi @klongleywood
I've reviewed the data you shared, and I am not sure it is the best suit as it currently is. The main issue is that the normalized data is just the total data divided in every case by country area, so the results of plotting that are proportionally the same as with total data, just in smaller units.
Data by intervention seems like a good a approach, but mangrove data still comes from a very reduced area (not sure about the others), so the magnitude doesn't help to highlight the message of its importance.
Perhaps we could use this data, but after discussing a more suitable normalization strategy.

**Comment by AngelArcones** [comment]
Hi @klongleywood , we have a follow-up to the previous comment.
Since the NCS mitigation data (total or normalized by area) was not the ideal too highlight the importance of mangrove conservation/restoration, we explored a little more into the data and came up with an alternative approach:
We used NCS data from [this paper](https://royalsocietypublishing.org/doi/10.1098/rstb.2019.0126), which I believe is similar to the one you provided. For a given country, we explored the proportion of the total mitigation potential that is considered as "cost-effective" (under US $100 per Mg CO2) on each intervention. This way, we can make a direct comparison between different interventions without the problem of different magnitudes (related to the size and landscape of each country), and have a common scale for all countries.
This is also useful because it will help highlighting the mangroves as an efficient mitigation investment, since more "direct" comparison (based on mitigation values per se) will almost always highlight the interventions on forests or agriculture (depending on the country) with the mangroves as a minor aspect.
Here I can show you an example of the data from Indonesia (let me know if you wish to see any other country):


![Screenshot 2022-02-24 at 09 21 36](https://user-images.githubusercontent.com/57727579/155491354-48f4e3ac-3a33-474b-8e0a-abe201313ce0.png)

Let us know your thought about this potential approach and any changes on it. If you decide to move on with this idea, there are some data / design aspects we would need to clarify:
- which data to use (if it's the one from the paper, we can take care of it. But maybe you have other data source you would like to use)
- should all categories be included, or can we group them by type? (e.g. both from mangrove together, one with all the forest-related combined... or maybe by action type)

**Comment by klongleywood** [comment]
@AngelArcones thank you for this!! I'm very optimistic that this approach will work, but let me just reach out to the rest of my team to confirm. I think the data from this paper is appropriate to use. I will ask about grouping too. Thanks for continuing to think creatively on this.

**Comment by klongleywood** [comment]
@AngelArcones I spoke to my team about this on Friday and they have some concerns about the data used in this paper. They are going to try to find data on the area of each intervention so that we can normalize by that rather than normalizing by total land area. If they can't track that down, we'll just forget about the comparative data and just show the stats for the mangrove interventions.

**Comment by AngelArcones** [comment]
Hi @klongleywood ,
following up on what we discussed about the fishing widget on ecosystem services, I share the different ideas I showed on our call for you to review and discuss with the team.
(designs are a bit rough, they were just too visualise some options)

Donut with values for each group:
![Screenshot 2022-03-11 at 16 05 33](https://user-images.githubusercontent.com/57727579/157893629-b84ff684-09a6-4b64-b198-1b719bc99e5a.png)

Boxes with total value and proportional %
![Screenshot 2022-03-11 at 16 05 43](https://user-images.githubusercontent.com/57727579/157893643-a87e7c6a-6ad9-4026-a16c-e3ccfbf82173.png)
 
Compared bars 
![Screenshot 2022-03-11 at 16 05 52](https://user-images.githubusercontent.com/57727579/157893690-27f07304-43d3-4942-8fe2-842effe9fdbd.png)

**Comment by klongleywood** [comment]
@AngelArcones I'm waiting for feedback from my team on the above design decision related to the fish enhancement. Should we put this question in another thread since it relates to another tool (not the climate and policy widget)?

For the climate and policy widget, I found data that normalizes some of the mitigation options by area of opportunity rather than total land area, so I think that will solve our problem, but there are a few nuances in the data that I also need feedback on. 

Our team meets next Friday, April 1, and I should have some decisions on these topics following that call.

**Comment by klongleywood** [comment]
@AngelArcones I got a suggestion today from the team doing this model. They asked if we could emulate the style of the graphic in Figure 4 of this paper https://link.springer.com/article/10.1007/s12237-021-00935-0/figures/4  where we have an little icon of a fish, a crab, and a shrimp (right now these are both lumped in to inverts but in the new data we will have them separated), and then the catch value listed next to the icon. In the paper, the values are expressed as percentages, but in GMW, we'd want these figures expressed as numbers of individuals (rounded values, expressed in the thousands or millions, or whatever is most appropriate). Can we do something like this?

**Comment by klongleywood** [comment]
@AngelArcones and fyi @lhilarides and @dhakelila 

I'm attaching a table that can be used to compare mangrove climate mitigation opportunities to other land use climate mitigation opportunities, where the denominator is the total area of opportunity in hectares -- this, I believe does the trick of communicating the high values of mangrove per hectare. The last column on mangrove restoration has data that will need to be updated once the mangrove restoration opportunity analysis is complete, but the values should be similar enough so that we can mock up this graphic to see how it looks, as the overall data structure won't change. Please let me know if you have any questions!

[Climate_Mitigation_per_Hectare_20220421.xlsx](https://github.com/Vizzuality/mangrove-atlas/files/8535368/Climate_Mitigation_per_Hectare_20220421.xlsx)

**Comment by klongleywood** [comment]
@AngelArcones we agree that the second design is best -- please move forward with that. The only requested change is to have the horizontal axis labels be horizontal. Thank you!

**Comment by klongleywood** [comment]
Hi @AngelArcones and @dhakelila -- happy to report I finally have updated NDC data for the climate and policy widget. 

As noted during the last call, we have a slight modification to the first blurb on the International Status Panel under NDC targets. Currently, it reads: Country's NDC target is xx mt CO2e/yr by xx year, which represents less than .01 of its GHG" It should now read as follows. I'm breaking up the clauses to specify how to deal with cases where information is missing, but these can all be lumped together for the first blurb. 

1. If Column J is populated, the first sentence should read:"[Country]'s NDC pledge contains [Column J]" If Column J is blank, leave this sentence out.  
2. For the second sentence, there are a few different ways this will read depending on data availability. Here are the possible cases:
     a. If Columns G, H, K, and L are all populated, the text should read: "The GHG target is a [Column G]% reduction from a baseline in [Column K] by target year [Column L]. This represents a reduction of [Column H] mtCO2e/yr."
     b. If Column G, K, and L are all populated but H is not, the text should read:  "The GHG target is a [Column G]% reduction from a baseline in [Column K] by target year [Column L]."
     c. If Column H, K, and L are all populated but G is not, the text should read: "The GHG target represents a reduction of [Column H] mtCO2e/yr from a baseline in [Column K] by target year [Column L]."
     d. If either Column K or L (or both) missing, just delete the associated clause(s) (i.e. "from a baseline in [Column K]" or "by target year [Column L] 

As before, we'd like to have a hyperlink linking to the NDC text (Column M). Finally, we'd like to have a "More info" hover or other functionality for including the NDC Blurb [Column O], perhaps right under that first blurb.

For the subsequent blurbs/sentences in that panel, note that I have added columns in this new spreadsheet so any Column linkages will just have to be adjusted (for example the sentence around the updated/first NDCs and whether they contain coastal and marine NBS will now come from Columns Q - T. 
[Copy of Copy of Climate_Policy_Tool_Data_20220707.csv](https://github.com/Vizzuality/mangrove-atlas/files/9066848/Copy.of.Copy.of.Climate_Policy_Tool_Data_20220707.csv)


I need to write up info button language for each of the panels so it might make sense to add a placeholder for an info hover icon or similar, and I will work on that draft metadata next week. 

I know this is a lot of info but hopefully not too confusing. Please let me know if you have questions!

**Comment by dhakelila** [comment]
Understood! Thank you!

**Comment by klongleywood** [comment]
@dhakelila @AngelArcones -- I've updated the data for the investible carbon panel of the tool in the attached table. Please let me know if you have questions.
[Investible_Blue_Carbon_Update_20220713.csv](https://github.com/Vizzuality/mangrove-atlas/files/9105433/Investible_Blue_Carbon_Update_20220713.csv)

**Comment by klongleywood** [comment]
I just deleted my last post on the investible carbon panel b/c it turns out I was using data from 2016 -- I'll have data from 2020 later this week

**Comment by klongleywood** [comment]
@AngelArcones @dhakelila  and FYI @lhilarides Here are the numbers for the investible carbon card on the climate and policy tool. I included instructions in the column headers but to summarize, Columns F, I, J, and K should be used to populate the pie chart values. columns E and G are labels for the two investible column values in the pie chart. There are also a number of countries that don't have investible carbon values calculated, so I added a column indicating whether a pie chart should be included (tagged with a Y) or not (tagged with a N). For the latter, instead of the pie chart, have text that says "Investible blue carbon values have not been calculated for this country"  Let me know if anything isn't clear. 

[Investible_Blue_Carbon_Update_20220725.csv](https://github.com/Vizzuality/mangrove-atlas/files/9184881/Investible_Blue_Carbon_Update_20220725.csv)

**Comment by dhakelila** [comment]
Hi @klongleywood if I understood it correctly, you are asking to include a logic to either show the donut or a sentence when there is no data for that country. That is not a big deal, but the project's logic so far is that when there is no data available for a country, we simply don't show that card. Do you think we can do the same in this case?

**Comment by klongleywood** [comment]
@dhakelila that's fine too!

**Comment by dhakelila** [comment]
Great! Thanks @klongleywood 

@AngelArcones , @aagm , @mluena . See my comment above regarding this widget and let's follow the logic of the project for consistency.

**Comment by klongleywood** [comment]
@AngelArcones @dhakelila fyi @lhilarides 

Hi all, I've written some draft info button language for the climate and policy tool. Please note that this is draft and will more than likely change after we have our review of this widget, but if possible, I think it would be helpful to include this draft language in the draft version that we show to reviewers so that they have something to react to, and can see how we're contextualizing the information. Would that be possible?

**1. Investible Blue Carbon** 

Estimates of mangrove forest area that can qualify for blue carbon financing that is financially sustainable over 30 years, based on prices of $5/ton and $10/ton based on [Zeng et al. (2021)](https://ap5.fas.nus.edu.sg/fass/geofd/zeng%20et%20al%202021%20return%20on%20investment%20mangrove%20blue%20carbon.pdf). These investible carbon areas are mangrove forests that can be protected through carbon financing. Areas of investible mangrove are defined as those under imminent threat of loss or decline if left unprotected by a conservation intervention. The estimate also accounts for carbon from above and below ground biomass, as well as soil carbon. Profitability of projects was based on average costs of project establishment and annual maintenance, weighted by the country’s GDP. Investible areas are rounded to the nearest 1,000

Estimates of mangroves in protected areas are based on the World Database of Protected Areas (July 2022). Calculations of remaining mangroves (i.e. mangroves that are not within protected areas and not considered investible) are based on GMW extent v 3.14. 
 
These data can be used to better understand, at a national scale, the potential of blue carbon finance that can be used towards climate mitigation goals, including nationally determined contributions specified under the Paris Climate Agreement.

**2. Emissions Mitigation by Area**

Estimates of emissions mitigation potential for mangrove interventions (e.g., protection or restoration) compared to other selected mitigation interventions. For these interventions, the total carbon mitigation opportunity is adjusted by the estimated area of the mitigation opportunity. The mitigation potential values used supplementary data from Roe et al. (2021), which aggregated mitigation potentials from a variety of individual and sectoral studies and datasets. For the purposes of this tool, we chose to include land-based mitigation measures that focused on forests and other ecosystems (e.g., grasslands, peatlands), and did not include values related to agriculture, bioenergy, or demand-side measures (e.g., food waste). Where multiple values for a single mitigation measure were reported (e.g., Reduce Deforestation), this tool used the average value reported in the supplementary data table. The source data contained both technical and cost-effective values. Here, only the technical values are reported. We updated the mitigation potential for mangrove restoration to newly calculated values from Worthington et al. (in prep). 

**3. International Status**

3a. Nationally determined contributions (NDCs) reflect a country’s proposed actions under the Paris Climate Agreement to cut emissions and adapt to climate change. Data used in this tool to describe a country’s NDC comes from [Global Climate Watch ](https://github.com/Vizzuality/mangrove-atlas/issues/climatewatchdata.org)(WRI 2022), or directly from the text of the country’s NDC.

3b. Information on whether a country’s NDC includes coastal and marine NBS (Nature-Based Solutions) comes from a policy brief ([Lecerf et al. 2021](https://ocean-climate.org/wp-content/uploads/2021/06/coastal-and-marine-ecosystem-2806.pdf)) developed in anticipation of the UNFCCC COP26 (United Nations Framework Convention on Climate Change Convention of the Parties, held in Glasgow in November 2021). NBS are actions to protect, sustainably manage and restore natural and modified ecosystems in ways that address societal challenges effectively and adaptively, to provide both human well-being and biodiversity benefits. In this context, mitigation measures aim to avoid and reduce emissions of GHGs into the atmosphere to prevent further warming. In the context of coastal and marine NBS, mitigation measures can include reducing loss of natural habitats that store carbon (e.g., mangroves and seagrasses), or restoring natural habitats that capture carbon.  Adaptation measures aim to alter behavior, systems, and ways of life to protect people, economies and the environment from climate change impact. In the context of coastal and marine NBS, these can include improve resilience of coastal ecosystems. Coastal and marine NBS provide important opportunities for responding to climate change and achieving national climate policy goals. The data presented here offer a chance to better understand where countries are already using coastal and marine NBS, and where there might be opportunities to enhance the use of NBS in future NDC revisions.

3c. The [wetlands supplement ](https://www.ipcc-nggip.iges.or.jp/public/wetlands/)is a methodology report adopted by the IPCC (International Panel on Climate Change), which contains guidelines for countries on improving their inventorying and reporting of GHGs from wetlands, with the goal of providing a pathway for countries to reduce agricultural emissions from peatlands and mangroves.

3d. [FRELs (Forest Reference Emission Level) ](https://www.forestcarbonpartnership.org/forest-reference-emission-levels)is a benchmark for emissions exclusively from deforestation and forest degradation. Setting a FREL is a necessary first step for countries to benefit from the REDD+ (Reducing Emissions from Deforestation and forest Degradation) program.  If mangroves are included in a country’s definition of a forest, they are eligible under the REDD+ program for climate credits.

**Comment by mluena** [comment]
@klongleywood Global Climate Watch redirects to an issue. I think it's worth moving that info to another place and linking there (maybe in wiki?)

**Comment by klongleywood** [comment]
@mluena  Oops -- copy and paste error on my part. We should link to this site: https://www.climatewatchdata.org/

**Comment by AngelArcones** [comment]
@klongleywood I've made a notebook showing the provisional data processing, implementation of the logic and phrasing for the **international status text widget**.
Feel free to review it from your side and suggest all the needed changes or corrections.
You can access the notebook here:
https://github.com/Vizzuality/mangrove-atlas/blob/data/international_status_notebook/data/notebooks/Lab/data_processing/widgets-v2/international_status_data_processing.ipynb

**Comment by klongleywood** [comment]
@AngelArcones thank you! A few minor edits: 

Sentence 1: . Countries with an NDC pledge should have an apostrophe s. For example: “Angola’s NDC pledge contains a GHG target”
Sentence 2: If there is only one target year, we should remove the /s. For example Antigua & Barbuda: The GHG target is a 25% reduction from a baseline in 1990 by target year/s 2030. If there are multiple years, the word should be plural (years not year/s)
Sentence 3: Country names should have an apostrophe s. For example “Australia’s updated NDC pledge”
Sentence 4: No changes needed

When I spoke to Clara a few weeks ago my understanding was that your team would be building a notebook demo similar to this https://public.tableau.com/app/profile/angel.arcones/viz/Mangroves_emission_mitigation/Story1 so that I could conduct testing with the climate and policy advisory group. Is that still happening? I am reviewing the widgets with this group on Wednesday August 17th, so please let me know if that is not going to happen so I can adjust my planning.

**Comment by klongleywood** [comment]
@AngelArcones  any updates on when this may be available for me to demo with our climate and policy group? Thanks!

**Comment by AngelArcones** [comment]
@klongleywood Sorry for the delay! We are now working on an interactive version for this to show and explore, and expect to have it ready on the next days.

**Comment by mluena** [comment]
@klongleywood in [this link](https://mangrove-atlas-git-feature-international-sta-aff369-vizzuality1.vercel.app/?map=eyJiYXNlbWFwIjoibGlnaHQiLCJ2aWV3cG9ydCI6eyJsYXRpdHVkZSI6MjAsImxvbmdpdHVkZSI6MCwiem9vbSI6MiwiYmVhcmluZyI6MCwicGl0Y2giOjB9fQ%3D%3D&widgets=eyJoaWdobGlnaHRlZF9wbGFjZXMiOnsiaXNDb2xsYXBzZWQiOnRydWUsImlzQWN0aXZlIjpmYWxzZX0sIm1hbmdyb3ZlX2V4dGVudCI6eyJpc0NvbGxhcHNlZCI6dHJ1ZSwiaXNBY3RpdmUiOnRydWV9LCJtYW5ncm92ZV9jb3ZlcmFnZSI6eyJpc0NvbGxhcHNlZCI6dHJ1ZSwiaXNBY3RpdmUiOmZhbHNlfSwibWFuZ3JvdmVfbmV0X2NoYW5nZSI6eyJpc0NvbGxhcHNlZCI6dHJ1ZSwiaXNBY3RpdmUiOmZhbHNlfSwibWFuZ3JvdmVfaW52ZXN0bWVudF9wb3RlbnRpYWwiOnsiaXNDb2xsYXBzZWQiOnRydWUsImlzQWN0aXZlIjpmYWxzZX0sIm1hbmdyb3ZlX2FsZXJ0cyI6eyJpc0NvbGxhcHNlZCI6dHJ1ZSwiaXNBY3RpdmUiOmZhbHNlfSwibWFuZ3JvdmVfc3BlY2llcyI6eyJpc0NvbGxhcHNlZCI6dHJ1ZSwiaXNBY3RpdmUiOmZhbHNlfSwibWFuZ3JvdmVfYmx1ZV9jYXJib24iOnsiaXNDb2xsYXBzZWQiOnRydWUsImlzQWN0aXZlIjpmYWxzZX0sIm1hbmdyb3ZlX3Byb3RlY3Rpb24iOnsiaXNDb2xsYXBzZWQiOnRydWUsImlzQWN0aXZlIjpmYWxzZX0sIm1hbmdyb3ZlX2hlaWdodCI6eyJpc0NvbGxhcHNlZCI6dHJ1ZSwiaXNBY3RpdmUiOmZhbHNlfSwibWFuZ3JvdmVfYmlvbWFzcyI6eyJpc0NvbGxhcHNlZCI6dHJ1ZSwiaXNBY3RpdmUiOmZhbHNlfSwiY29uc2VydmF0aW9uX2hvdHNwb3RzIjp7ImlzQ29sbGFwc2VkIjp0cnVlLCJpc0FjdGl2ZSI6ZmFsc2V9LCJtYW5ncm92ZV9hY3Rpdml0eSI6eyJpc0NvbGxhcHNlZCI6dHJ1ZSwiaXNBY3RpdmUiOmZhbHNlfSwibWFuZ3JvdmVfcmVzdG9yYXRpb24iOnsiaXNDb2xsYXBzZWQiOnRydWUsImlzQWN0aXZlIjpmYWxzZX0sIm1hbmdyb3ZlX2VtaXNzaW9uc19taXRpZ2F0aW9uIjp7ImlzQ29sbGFwc2VkIjp0cnVlLCJpc0FjdGl2ZSI6ZmFsc2V9LCJtYW5ncm92ZV9pbnRlcm5hdGlvbmFsX3N0YXR1cyI6eyJpc0NvbGxhcHNlZCI6dHJ1ZSwiaXNBY3RpdmUiOmZhbHNlfX0%3D) you can play with international status widget. The sentences are created according with the criteria you gave us. 

Just for testing -> When you hover each sentence you can see the cursor reacting, if you click on the sentence you will get an input for you to put a custom sentence (I added this just in case you wanted to try with any other text, longer or shorter). I will remove that after your testing

**Comment by mluena** [comment]
@klongleywood in [this link](https://mangrove-atlas-git-feature-international-sta-aff369-vizzuality1.vercel.app/?map=eyJiYXNlbWFwIjoibGlnaHQiLCJ2aWV3cG9ydCI6eyJsYXRpdHVkZSI6MjAsImxvbmdpdHVkZSI6MCwiem9vbSI6MiwiYmVhcmluZyI6MCwicGl0Y2giOjB9fQ%3D%3D&widgets=eyJoaWdobGlnaHRlZF9wbGFjZXMiOnsiaXNDb2xsYXBzZWQiOnRydWUsImlzQWN0aXZlIjpmYWxzZX0sIm1hbmdyb3ZlX2V4dGVudCI6eyJpc0NvbGxhcHNlZCI6dHJ1ZSwiaXNBY3RpdmUiOnRydWV9LCJtYW5ncm92ZV9jb3ZlcmFnZSI6eyJpc0NvbGxhcHNlZCI6dHJ1ZSwiaXNBY3RpdmUiOmZhbHNlfSwibWFuZ3JvdmVfbmV0X2NoYW5nZSI6eyJpc0NvbGxhcHNlZCI6dHJ1ZSwiaXNBY3RpdmUiOmZhbHNlfSwibWFuZ3JvdmVfaW52ZXN0bWVudF9wb3RlbnRpYWwiOnsiaXNDb2xsYXBzZWQiOnRydWUsImlzQWN0aXZlIjpmYWxzZX0sIm1hbmdyb3ZlX2FsZXJ0cyI6eyJpc0NvbGxhcHNlZCI6dHJ1ZSwiaXNBY3RpdmUiOmZhbHNlfSwibWFuZ3JvdmVfc3BlY2llcyI6eyJpc0NvbGxhcHNlZCI6dHJ1ZSwiaXNBY3RpdmUiOmZhbHNlfSwibWFuZ3JvdmVfYmx1ZV9jYXJib24iOnsiaXNDb2xsYXBzZWQiOnRydWUsImlzQWN0aXZlIjpmYWxzZX0sIm1hbmdyb3ZlX3Byb3RlY3Rpb24iOnsiaXNDb2xsYXBzZWQiOnRydWUsImlzQWN0aXZlIjpmYWxzZX0sIm1hbmdyb3ZlX2hlaWdodCI6eyJpc0NvbGxhcHNlZCI6dHJ1ZSwiaXNBY3RpdmUiOmZhbHNlfSwibWFuZ3JvdmVfYmlvbWFzcyI6eyJpc0NvbGxhcHNlZCI6dHJ1ZSwiaXNBY3RpdmUiOmZhbHNlfSwiY29uc2VydmF0aW9uX2hvdHNwb3RzIjp7ImlzQ29sbGFwc2VkIjp0cnVlLCJpc0FjdGl2ZSI6ZmFsc2V9LCJtYW5ncm92ZV9hY3Rpdml0eSI6eyJpc0NvbGxhcHNlZCI6dHJ1ZSwiaXNBY3RpdmUiOmZhbHNlfSwibWFuZ3JvdmVfcmVzdG9yYXRpb24iOnsiaXNDb2xsYXBzZWQiOnRydWUsImlzQWN0aXZlIjpmYWxzZX0sIm1hbmdyb3ZlX2VtaXNzaW9uc19taXRpZ2F0aW9uIjp7ImlzQ29sbGFwc2VkIjp0cnVlLCJpc0FjdGl2ZSI6ZmFsc2V9LCJtYW5ncm92ZV9pbnRlcm5hdGlvbmFsX3N0YXR1cyI6eyJpc0NvbGxhcHNlZCI6dHJ1ZSwiaXNBY3RpdmUiOmZhbHNlfX0%3D) you can play with international status widget. The sentences are created according with the criteria you gave us. 

Just for testing -> When you hover each sentence you can see the cursor reacting, if you click on the sentence you will get an input for you to put a custom sentence (I added this just in case you wanted to try with any other text, longer or shorter). I will remove that after your test

**Comment by klongleywood** [comment]
Thanks @mluena! Any chance you can also make the Investible Blue Carbon widget available on staging again? At that point, we'll be ready to test the whole suite of climate and policy tools

**Comment by mluena** [comment]
Do you want me to include it in the link I've just sent to you, so you will have both of them together?

**Comment by klongleywood** [comment]
@mluena  yes please!

**Comment by mluena** [comment]
@klongleywood done! I removed the rest of the widgets and left the widgets related with this one in "distribution and change"category so you don't have to keep changing category. Once it's approved I'll put the rest back in place

**Comment by klongleywood** [comment]
@mluena -- I just went to that link and now it seems to only be showing the Mangrove Emissions Mitigation and the Blue Carbon widgets. The 4 widgets that I need to have my team review under the Climate and Policy category are. 

Mangrove Emissions and Mitigaton
International Status
Investible Blue Carbon
Blue Carbon

If we could put all of these in the same staging environment, that would be very helpful. Thanks!

**Comment by mluena** [comment]
ok, let me take a look because those four are exactly the ones I left, so if you are not seeing all of them I need to check where the problem is. I let you know as soon as I fix it

**Comment by klongleywood** [comment]
@mluena thanks -- I'm now seeing all but investible blue carbon

**Comment by mluena** [comment]
@klongleywood ok, I see the problem now. For some of them there is no data in the API (we agreed to improve the structure so we cleared the endpoints). For emissions mitigation for example, you are seeing the "placeholder" I put just to check the visualisation. The only one showing real data is investment potential. I'll let you know when data is ready so you are able to see the 4th of them together for the test

**Comment by klongleywood** [comment]
@mluena ok, thank you! Let me know if there's anything else you need from me.

**Comment by klongleywood** [comment]
@mluena  - just checking in -- when do you think this will be ready? Thanks!

**Comment by mluena** [comment]
We can´t update the endpoints because Alicia is out this week. Angel and I
are going to try to prepare the rest of the data for tomorrow so it can
serve you for testing even though it doesn't go through the API

**Comment by mluena** [comment]
@klongleywood I have updated the branch with the data straight away in the platform so you can check the 4 widgets for different locations

**Comment by klongleywood** [comment]
@mluena thanks! I will start review with my team. One thing I see missing is the information about the FRELs -- See slide 2 in the ppt mock-up provided on December 16th in the above thread. The text about FRELs and whether or not a mangrove is considered a forest or a wetland was originally its own widget, but we asked to have that text moved under the International Status heading. In the CSV, data feeding into this is Columns U and V.

**Comment by AngelArcones** [comment]
@klongleywood Regarding the sentence for the FREL data, in the presentation it currently displays as (highlighting the potentially dynamic parts):

`Indonesia’s` `2017` FREL is `48.9` Mt CO₂e/yr (Indonesia’s mangroves are considered `[Forest/Wetland]` according to `xxx`). 
However, we seem to lack information for year (is it always the same?) and also de source for the "according to". Is this information available, or should we rephrase the sentence to skip those parts?

**Comment by klongleywood** [comment]
@AngelArcones -- I'm linking to an updated spreadsheet that has the FREL in the last column. And you're right, we never got the "according to xxx" data, so just leave out that part of the sentence. Thanks!
[Copy of Copy of Climate_Policy_Tool_Data_20220901.csv](https://github.com/Vizzuality/mangrove-atlas/files/9473081/Copy.of.Copy.of.Climate_Policy_Tool_Data_20220901.csv)

**Comment by klongleywood** [comment]
@dhakelila here are the issues that emerged on the initial review call -- I expect to have the rest of the review wrapped up by Friday. 

**General**
1. Issue: no visible layers on the map when the Climate Policy dashboard is opened. 
Solution: switch on the Mangrove Blue Carbon layer when the “Climate & Policy” dashboard is opened

**Widget 1 Investible Blue Carbon**

1. Change title of widget to "Carbon market Potential"
2. Change data description to "Extent of investible mangrove carbon (ha) is..."  There also needs to be a way for users to have that sentence describe the values at $10/ton. Can we have a drop down menu where they can choose the value? 
3. Change donut labels from Carbon_10 and Carbon_5 to:  Additional Investible Blue Carbon @ $10/ ton, Investible Blue Carbon @ $5/ton. Protected should read “Area of Mangrove in Protected Areas” and Remaining should read “Remaining mangrove”
4. The widget text should read: “The extent of investible blue carbon (ha) in [Country] is…”
5. There needs to be a way for users to see the total area in hectares (not just the percentages) when they hover over each of the pieces of the donut. Suggest that for the two carbon values it should be the total values with the standard errors (e.g., 98,900 (+- 2800). 
6. Move to bottom of widget list (as this is the most potentially controversial widget we don't want it front and center)

**Widget 2. Mangrove Emissions Mitigation** -- I can't currently see this on staging -- can you return it to the dedicated staging site while we review it?

1. Colors need more contrast
2. Double check the units. Should be tCO2e/ha
3. If possible, re-order the legend to put mangroves first on the list
4. I can't see this widget now, but last time I looked, the numbers for mangroves looked off. When we supply this widget with updated data, let's make sure it's pulling from the correct column

Widget 3. Mangrove International Status

1. For the second sentence (starting with “The GHG target….) the unit should be tCO2/yr 
2. When reporting the numbers of the FRELs, insert comma separator 
3. In the third sentence, there needs to be an apostrophe after the country name
4. Add the hyperlink to the source of the NDC pledge 
5. We need to specify whether the NDC pledge is for mitigation, adaptation, or both, where applicable -- this is in the spreadsheet
6. Take out the part of the sentence that refers to the baseline year. Confusing and it changes frequently. So, should just say 
"The GHG target is a % reduction by target year 2030."
7. If there is a null value for whether the mangrove is considered a wetland or a forest, remove that part of the sentence. 
8. If a country hasn’t implemented the wetlands supplement, say that “There is no information as to whether this country has implemented the wetlands supplement”

**Comment by dhakelila** [comment]
Great, thanks for this feedback. We'll prioritise the rest of the items so we wait for the rest of the feedback on Friday but we'll start if time permits.

**Comment by dhakelila** [comment]
_Issue: no visible layers on the map when the Climate Policy dashboard is opened.
Solution: switch on the Mangrove Blue Carbon layer when the “Climate & Policy” dashboard is opened_

This changes the logic of the rest of the widgets and might be misleading. I think the behaviour you are looking for is something similar to restoration, but it's not the same because, on the Restoration, we created a widget with many widgets. We could eventually do this with Policy and Climate, but not now. 

Changes on the Widget 1 Investible Blue Carbon done.

**Comment by dhakelila** [comment]
Changes on the Widget 1 Investible Blue Carbon done.

**Comment by dhakelila** [comment]
@klongleywood , @klongleywood please, check the Emissions Mitigation color palette proposed. I'm not sure if we'd like to have all the non-mangroves similar tone (like different greens but all of them green)



![Screenshot 2022-09-19 at 12 32 14](https://user-images.githubusercontent.com/6608294/190999132-3ca6b633-3cd8-4435-a8ed-cfb0adc43309.png)

**Comment by dhakelila** [comment]
Notes from the meeting: Mangroves in green and other in orange-purple. Less "pastel".

**Comment by klongleywood** [comment]
fyi @dhakelila @lhilarides 

On the international status widget, there are a few issues:

- The following change specified above has not been implemented: If a country hasn’t implemented the wetlands supplement, say that “There is no information as to whether this country has implemented the wetlands supplement”
- Another change that hasn't been implemented yet is the following: For countries where it specified, We need to specify whether the NDC pledge is for mitigation, adaptation, or both, where applicable -- this is in the spreadsheet columns P&Q. So, for example, for Brunei, the text should read "Brunei's updated NDC pledge includes coastal and marine NBS for mitigation and adaption" If it yes for one and no for another it should say "....includes coastal and marine NBS for mitigation but not adaptation". If there is no data leave this sentence out. 
- The units should be MtCO2/yr  (not just small t -- sorry, just noticed this one)
- I posted in the other thread about updated info button language for this, and the carbon market potential widget

- The carbon market potential is currently showing at the global level -- it should not be visible globally, only for countries where there are data.

**Comment by dhakelila** [comment]
@klongleywood changes ready in staging. Let us know if you want us to deploy to production and we'll do it asap. 

cc/ @mluena

**Comment by klongleywood** [comment]
@dhakelila @mluena fyi @lhilarides 

Almost ready, but still a few issues. 

- Thanks for adding the additional information about adaptation/mitigation. There is an apostrophe that shows up at the end of the sentence (see Angola for example). Can you remove that?
- The units for the second sentence are still incorrect. Should be MtCO2e/yr

-In the Carbon market potential, I don't think the values of the donut are reflecting the correct values for the Additional Investible Blue carbon at $10/ton. As specified in the table I sent, these should be pulling from Column I because it reflects the additional value from carbon at $10/ton. 
This is different than the text about Blue Carbon at $10/ton which pulls from Column G (this is currently correct).

**Comment by mluena** [comment]
Both widgets have been updated, let me know if you want me to show "International Status" in prod @klongleywood

**Comment by klongleywood** [comment]
@mluena thank you! Feel free to push to prod

**Comment by dhakelila** [comment]
Emissions widget ready for your revision on staging.

![Screenshot 2022-09-26 at 13 13 23](https://user-images.githubusercontent.com/6608294/192262592-2f250d62-cb80-4dc7-a474-b414c55e9e77.png)
